pkg/cli-utils.go

```diff
@@ -110,22 +110,47 @@ func NewCustomResourceDefinition(config Config) *extensionsobj.CustomResourceDef
 	return crd
 }
 
-func MarshallCrd(crd *extensionsobj.CustomResourceDefinition, outputFormat string) {
-	jsonBytes, err := json.MarshalIndent(crd, "", "    ")
+func MarshallCrd(crd *extensionsobj.CustomResourceDefinition, outputFormat string) error {
+	jsonBytes, err := json.Marshal(crd)
```

Comment:
Will this be simpler?

```
r := *crd
r.Status = CustomResourceDefinitionStatus{}
jsonBytes, err := json.MarshalIndent(r, "", "    ")
```

Reply:
Unfortunately no, the CustomResourceDefinition object has a non pointer CustomResourceDefinitionStatus that has non pointer values, which always make it non-empty, therefore the omitempty is not triggered.

Reply:
I see what you mean. lgtm .